lib/ansible/modules/cloud/amazon/dynamodb.py

```diff
@@ -0,0 +1,359 @@
+    returned: success
+    type: dict
+    sample:
+        {
+            "bank": {
+                "S": "hsbc"
+            },
+            "quantity": {
+                "N": "1000"
+            },
+            "person": {
+                "S": "ochoa"
+            }
+        }
+'''
+
+import ansible.module_utils.ec2
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.ec2 import boto_exception
+
+
+try:
+    import boto3
+    import botocore
+    HAS_BOTO3 = True
+except ImportError:
+    HAS_BOTO3 = False
+
+
+def get(connection, table, primary_key, projection_expression, result):
+
+    if projection_expression:
+        response = connection.get_item(
+            TableName=table,
+            Key=primary_key,
+            ProjectionExpression=projection_expression
+        )
+    else:
+        response = connection.get_item(
+            TableName=table,
+            Key=primary_key
+        )
+
+    if 'Item' in response:
+        result['item'] = response['Item']
+    else:
+        result = dict(failed=True, message='No item found')
+
+    return result
+
+
+def put(connection, table, item, result):
+
+    connection.put_item(
+        TableName=table,
+        Item=item,
+    )
+
+    result = dict(changed=True, message='Item added')
+
+    return result
+
+
+def update(connection, table, primary_key, update_expression,
+           condition_expression, expression_attribute_names,
+           expression_attribute_values, result):
+
+    if condition_expression:
+        connection.update_item(TableName=table, Key=primary_key, UpdateExpression=update_expression,
+                               ConditionExpression=condition_expression, ExpressionAttributeValues=expression_attribute_values)
+    elif expression_attribute_names:
+        connection.update_item(TableName=table, Key=primary_key, UpdateExpression=update_expression,
+                               ExpressionAttributeValues=expression_attribute_values, ExpressionAttributeNames=expression_attribute_names)
+    elif update_expression and expression_attribute_values:
+        connection.update_item(TableName=table, Key=primary_key, UpdateExpression=update_expression,
+                               ExpressionAttributeValues=expression_attribute_values)
+    else:
+        connection.update_item(
+            TableName=table, Key=primary_key, UpdateExpression=update_expression)
+
+    result = dict(changed=True, message='Item updated')
+    return result
+
+
+def delete(connection, table, primary_key, condition_expression,
+           expression_attribute_values, result):
+
+    connection.delete_item(TableName=table, Key=primary_key, ConditionExpression=condition_expression,
+                           ExpressionAttributeValues=expression_attribute_values)
+
+    result = dict(changed=True, message='Item deleted')
+
+    return result
+
+
+def main():
+    argument_spec = ansible.module_utils.ec2.ec2_argument_spec()
```

Comment:
Merging with `ec2_argument_spec()` is automatically handled by AnsibleAWSModule.